iree/testing/BUILD

```diff
@@ -38,6 +40,16 @@ cc_library(
     ],
 )
 
+# gtest library has implicit thread dependency from absl.
+iree_cmake_extra_content(
+    content = """
+if (NOT ${IREE_ENABLE_THREADING})
```

Comment:
link the TODO (though this one may be hard to change since it is indirect through gtest) + style nit
```suggestion
# TODO(#3848): gtest library has implicit thread dependency from absl.
iree_cmake_extra_content(
    content = """
if(NOT ${IREE_ENABLE_THREADING})
```

Reply:
done.